test/new/db/tools/rabin2

```diff
@@ -625,7 +633,19 @@ Version need section '.gnu.version_r' contains 1 entries:
  Addr: 0x08048250  Offset: 0x00000250  Link to section: 6 (.dynstr)
   0x00000000: Version: 1  File: libc.so.6  Cnt: 1
   0x00000010:   Name: GLIBC_2.0  Flags: none Version: 2
-EOF
+Section to Segment mapping:
+Segment      Section
+--------------------
+PHDR         
+INTERP       .interp 
+LOAD0        .interp .note.ABI_tag .note.gnu.build_id .gnu.hash .dynsym .dynstr .gnu.version .gnu.version_r .rel.dyn .rel.plt .init .plt .text .fini .rodata .eh_frame_hdr .eh_frame 
+LOAD1        .init_array .fini_array .jcr .dynamic .got .got.plt .data .bss 
+DYNAMIC      .dynamic 
+NOTE         .note.ABI_tag .note.gnu.build_id 
+GNU_EH_FRAME .eh_frame_hdr 
+GNU_STACK    
+ehdr         
+<<EOF
```

Comment:
this should be `EOF` not `<<EOF`. If you look at all the other tests you see they use `EOF`